### ISSUE TITLE
Same icustay_id linked to 2 different hadm_ids in inputevents_cv?

### ISSUE THREAD

**Issue by matteobonvini** [MIT-LCP/mimic-code#120]
Dear all,

First, thank you for letting us work on this great database! I have noticed that in inputevents_cv there are a few cases (I think 52) where the same icustay_id is linked to 2 different hadm_ids (same subject_id). For example:

`select distinct hadm_id, icustay_id
from mimiciii.inputevents_cv where icustay_id = 201005`

Are these cases ok exceptions?

Thank you very much!
Matteo

**Comment by alistairewj** [comment]
Thanks - this has been fixed in the new version that will be released (v1.4). Currently, data is duplicated and assigned to two `ICUSTAY_ID`s if the patient is discharged/readmitted to *both* the hospital and the ICU within 12 hours, e.g.:

```sql
select subject_id, hadm_id, icustay_id
, charttime, itemid, amount, amountuom 
from inputevents_cv
where icustay_id = 201005
and charttime > to_timestamp('2169-12-04 12:00:00','YYYY-MM-DD HH24:MI:SS')
and charttime < to_timestamp('2169-12-04 18:00:00','YYYY-MM-DD HH24:MI:SS')
order by charttime;
```

gives:

subject_id | hadm_id | icustay_id |      charttime      | itemid | amount | amountuom 
------------ | --------- | ------------ | --------------------- | -------- | -------- | -----------
31755 |  105833 |     201005 | 2169-12-04 13:00:00 |  30403 |     80 | ml
31755 |  105833 |     201005 | 2169-12-04 15:00:00 |  30403 |     40 | ml
31755 |  105833 |     201005 | 2169-12-04 16:00:00 |  30403 |     40 | ml

.. and using the other `ICUSTAY_ID`:

```sql
select subject_id, hadm_id, icustay_id
, charttime, itemid, amount, amountuom 
from inputevents_cv
where icustay_id = 241770
and charttime > to_timestamp('2169-12-04 12:00:00','YYYY-MM-DD HH24:MI:SS')
and charttime < to_timestamp('2169-12-04 18:00:00','YYYY-MM-DD HH24:MI:SS')
order by charttime;
```

gives the same data (duplicated):

subject_id | hadm_id | icustay_id |      charttime      | itemid | amount | amountuom 
------------ | --------- | ------------ | --------------------- | -------- | -------- | -----------
31755 |  105833 |     241770 | 2169-12-04 13:00:00 |  30403 |     80 | ml
31755 |  105833 |     241770 | 2169-12-04 15:00:00 |  30403 |     40 | ml
31755 |  105833 |     241770 | 2169-12-04 16:00:00 |  30403 |     40 | ml

You'll have to play around this for now. The issue only affects 37 of the 60,000+ `ICUSTAY_ID`s though, so shouldn't be too big of a deal.

**Comment by alistairewj** [comment]
This has been fixed in v1.4.
